vsts-fullbuild-ubuntu-vm/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Ubuntu VM with cross-platform SDKs and Team Services agent",
```

Comment:
It would be good to have the names reflect the full build and min build? right now they seem pretty similar